From dfd4943ca35bd98a34008d146d820b969a305dee Mon Sep 17 00:00:00 2001
From: Nicholas Devenish <ndevenish@gmail.com>
Date: Thu, 16 Jan 2025 12:49:06 +0000
Subject: [PATCH] Change cadquery dependency to ocp dependency

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 839d309..58d3420 100755
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,8 +5,8 @@ readme = "readme-pypi.md"
 version = "0.3.4"
 requires-python = ">=3.9"
 dependencies = [
-    "cadquery-ocp >= 7.7.0, <7.8",
-    "svgelements >= 1.9.1, <2",
+    "ocp >= 7.7.0, <7.8",
+    "svgelements >= 1.9.1,<2",
 ]
 
 
-- 
2.47.0

